GliaWidgets/Window/GliaViewController.swift

```diff
@@ -0,0 +1,107 @@
+import UIKit
+
+public enum GliaViewControllerEvent {
+    case minimized
+    case maximized
+}
+
+public protocol GliaViewControllerDelegate: AnyObject {
+    func event(_ event: GliaViewControllerEvent)
+}
+
+class GliaViewController: UIViewController {
+    var bubbleKind: BubbleKind = .userImage(url: nil) {
+        didSet { bubbleWindow?.bubbleKind = bubbleKind }
+    }
+
+    private enum State {
+        case maximized
+        case minimized
+    }
+
+    private var state: State = .maximized
+    private weak var delegate: GliaViewControllerDelegate?
+    private var gliaNavigationController: UINavigationController!
```

Comment:
Let's avoid using implicitly unwrapped optional (if possible). Probably we could use optional or lazy variable instead.
Looks like we are using `gliaNavigationController` just to retain the `navigationController`, I could be wrong, but does `addChild` doesn't retain it? If so, we could delete this var at all.

Reply:
I think indeed the best way would be to remove it, you're right